ts-client/src/dlmm/index.ts

```diff
@@ -3072,14 +3150,50 @@ export class DLMM {
       .postInstructions(postInstructions)
       .transaction();
 
+    const setCUIx = await getEstimatedComputeUnitIxWithBuffer(
+      this.program.provider.connection,
+      removeLiquidityTx.instructions,
+      user,
+      DEFAULT_REMOVE_LIQUIDITY_CU
+    );
+
+    removeLiquidityTx = await this.program.methods
```

Comment:
feel a bit duplicated with the code above, is there a way to refactor that?

Reply:
Fixed, it actually create issue and causes test to fail